vaadin-date-time-picker-flow-demo/src/main/java/com/vaadin/flow/component/datetimepicker/demo/DateTimePickerView.java

```diff
@@ -62,6 +63,20 @@ private void basicDemo() {
         addCard("Basic usage", div);
     }
 
+    private void createMinAndMaxDateTimePicker() {
+        // begin-source-example
+        // source-example-heading: Date time picker with min and max
+        DateTimePicker dateTimePicker = new DateTimePicker();
+
+        LocalDateTime now = LocalDateTime.now();
+
+        dateTimePicker.setMin(now);
+        dateTimePicker.setMax(now);
+        // end-source-example
+
+        addCard("Date time picker with min and max", dateTimePicker);
```

Comment:
1. "Date time picker with" is a bit redundant in the heading (`source-example-heading` comment and `addCard` line). I'd use just "Min and max".

2. This is not a very good demo imo since it's not a very realistic use case to set both to the same precise value so there is only one valid value possible. Here's my proposal for a better demo:

```java
    private void createMinAndMaxDateTimePicker() {
        // begin-source-example
        // source-example-heading: Min and max
        DateTimePicker dateTimePicker = new DateTimePicker();

        LocalDate today = LocalDate.now();
        LocalDateTime min = LocalDateTime.of(today, LocalTime.MIN);
        LocalDateTime max = LocalDateTime.of(today.plusWeeks(1), LocalTime.MAX);

        dateTimePicker.setMin(min);
        dateTimePicker.setMax(max);
        // end-source-example

        addCard("Min and max", dateTimePicker);
    }
```

Reply:
Updated.